thoth/storages/inspections.py

```diff
@@ -31,3 +35,23 @@ class InspectionResultsStore(ResultStorageBase):
     def get_document_id(cls, document: dict) -> str:
         """Get id under which the given document will be stored."""
         return document["inspection_id"]
+
+    def retrieve_document_ids(cls) -> List[str]:
+        """Retrieve a list of stored document from S3."""
+        return list(cls.get_document_listing())
```

Comment:
This functionality is already provided by base class:

https://github.com/thoth-station/storages/blob/a214289a3c11629e40113bd3a91eeffa0120939b/thoth/storages/result_base.py#L82-L84

Which will call ceph adapter and it will return desired listing:

https://github.com/thoth-station/storages/blob/a214289a3c11629e40113bd3a91eeffa0120939b/thoth/storages/ceph.py#L60-L63